Tests/CMakeLists.txt

```diff
@@ -1,13 +1,5 @@
 enable_testing()
 
-if (USE_INCLUDED_GTEST)
```

Comment:
There is an opton
option(USE_INCLUDED_GTEST "Used included gtest" ON)
in the main CMakeLists.txt https://github.com/3MFConsortium/lib3mf/blob/develop/CMakeLists.txt#L27C22-L27C23 .
This shoubd be removed, too.